linodecli/api_request.py

```diff
@@ -162,7 +175,13 @@ def _build_filter_header(
         filter_list.extend(new_filters)
 
     if len(filter_list) > 0:
-        return json.dumps({"+and": filter_list})
+        if order_by is None:
+            return json.dumps({"+and": filter_list})
+        return json.dumps(
+            {"+and": filter_list, "+order_by": order_by, "+order": order}
+        )
+    if order_by is not None:
+        return json.dumps({"+order_by": order_by, "+order": order})
 
     return None
```

Comment:
```suggestion
    result = {}

    if len(filter_list) > 0:
        result["+and"] = filter_list

    if order_by is not None:
        result["+order_by"] = order_by
        result["+order"] = order

    return json.dumps(result) if len(result) > 0 else None
```

Reply:
Will likewise change this one too, thanks for the suggestions!